packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -559,10 +534,12 @@ describe('Meter', () => {
         assert.doesNotThrow(() => boundValueRecorder.record(10));
       });
 
-      it('should not accept negative values by default', async () => {
-        const valueRecorder = meter.createValueRecorder('name');
+      it('should not set the instrument data when disabled', async () => {
+        const valueRecorder = meter.createValueRecorder('name', {
+          disabled: true,
+        }) as ValueRecorderMetric;
         const boundValueRecorder = valueRecorder.bind(labels);
-        boundValueRecorder.record(-10);
+        boundValueRecorder.record(10);
```

Comment:
This diff makes things confusing but that test case was not touched, its just a shifting of code.